components/Tabs/src/TabBarView/MDCTabBarView.h

```diff
@@ -14,6 +14,30 @@
 
 #import <UIKit/UIKit.h>
 
+@class MDCTabBarView;
+
+/**
+ Delegate protocol for MDCTabBarView. Clients may implement this protocol to receive notifications
+ of selection changes by user action in the tab bar.
+ */
+@protocol MDCTabBarViewDelegate <UIScrollViewDelegate>
```

Comment:
The delegate should not inherit from `UIScrollViewDelegate` since that would restrict both a client's ability to spread the decision making and also restrict refactoring of the TabBarView in the future.

Reply:
Done.